tests/test_art.crap

```diff
@@ -0,0 +1,36 @@
+from importlib import import_module
```

Comment:
Does this workfile want to be in the merge? Probs needs a better name if so.

Reply:
oh man...that should not be in there...let me remove